meta.yml

```diff
@@ -3,17 +3,17 @@ kit:
   link: https://github.com/qubell-bazaar/broadleaf-starter-kit
   applications:
     - name: "Broadleaf"
-      manifest: https://raw.github.com/qubell-bazaar/broadleaf-starter-kit/1.3-39p/broadleaf-starter-kit.yml
+      manifest: https://raw.github.com/qubell-bazaar/broadleaf-starter-kit/gce/broadleaf-starter-kit.yml
```

Comment:
this should not rely on work.
default should stay ec2, if you have to add another behavior let it be another install button.